lib/users.js

```diff
@@ -38,35 +38,22 @@ users.sendPasswordResetEmail = async (userAccount) => {
 };
 
 users.signUp = async (userInfo) => {
-  try {
-    await schema.validate(userInfo);
-  } catch (err) {
-    throw new Error(err);
-  }
-
-  const { email } = userInfo;
-  const { Accounts } = db.getModels();
-  const userAccount = await Accounts.findOne({
```

Comment:
I also took this part out because `Accounts.create` function in 49th(before 64th) line also check unique-validation.